src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -481,22 +452,46 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
                   (  drhoDT * dTdzTop(k+1) &
                    + drhoDS * dSdzTop(k+1) )
 
+               ! set taper of slope ('F' function from Danabasoglu and McWilliams 95)
+               slopeTaperUp = 0.5_RKIND*(1.0 - tanh((abs(slopeTriadUp(k,iCellSelf,iEdge)) - &
+                  config_Redi_half_slope)/config_Redi_half_width))
+               slopeTaperDown = 0.5_RKIND*(1.0 - tanh((abs(slopeTriadDown(k,iCellSelf,iEdge)) - &
+                  config_Redi_half_slope)/config_Redi_half_width))
+
+               zEdge = 0.5_RKIND*(zMid(k,cell1) + zMid(k,cell2))
+               verticalDisplacement = rediRossbyRadius(iEdge)*abs(slopeTriadUp(k,iCellSelf,iEdge))
+               rValue = zEdge / verticalDisplacement
+
+               sfcTaperUp = 0.5_RKIND*(1.0_RKIND + sin(3.141592_RKIND*(rValue - 0.5_RKIND)))
+               sfcTaperUp = max(0.0_RKIND, min(sfcTaperUp, 1.0_RKIND))
+
+               verticalDisplacement = rediRossbyRadius(iEdge)*abs(slopeTriadDown(k,iCellSelf,iEdge))
+               rValue = zEdge / verticalDisplacement
+
+               sfcTaperDown = 0.5_RKIND*(1.0_RKIND + sin(3.141592_RKIND*(rValue - 0.5_RKIND)))
+               sfcTaperDown = max(0.0_RKIND, min(sfcTaperDown, 1.0_RKIND))
+
+               ! LPV move vertical lines up to not limit the vertical mixing term
+               ! Griffies 1998 eqn 34
+               k33(k  ,iCell) = k33(k  ,iCell) + slopeTaperUp * sfcTaperUp * &
+                              areaEdge*dzTop(k)*slopeTriadUp(k,iCellSelf,iEdge)**2
+               k33(k+1,iCell) = k33(k+1,iCell) + slopeTaperDown * sfcTaperDown * &
```

Comment:
@vanroekel, these k33 lines were moved up in the last commit, but they still have the taper coefficients, same as slopeTriadUp and slopeTraidDown. The plan was to *not* taper k33, right? So shouldn't the slope taper variables not appear in the above lines?